src/main/java/com/dongtronic/diabot/platforms/discord/commands/nightscout/NightscoutCommand.kt

```diff
@@ -190,16 +196,25 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand(category, nu
                         return
 
                     "$domain/api/v1/"
+                }
+                // Try to get nightscout data from username/nickname, otherwise just try to get from hostname
+                else if (namedMembers.size == 1) {
+                    val user = namedMembers[0].user
+                    if (getNightscoutPublic(user)) {
+                        getUserDto(user, userDTO)
+                        getNightscoutHost(user) + "/api/v1/"
+
+                    } else {
```

Comment:
perhaps send a message saying there was a user found but their data is private. 
Currently, the following happens:

assume: user with name `testuser` that has NS data set to private, no nightscout instance exists with the hostname `testuser.herokuapp.com`

result: :cry: reaction with no other explanation.

Reply:
Should be better now, it'll still try to get with the hostname if the data is private but it finds a Nightscout instance with the url.